core/services/job/orm.go

```diff
@@ -186,27 +186,27 @@ func (o *orm) CreateJob(jb *Job, qopts ...pg.QOpt) error {
 				}
 			}
 
+			if jb.OCROracleSpec.EVMChainID == nil {
+				// If unspecified, assume we're creating a job intended to run on default chain id
+				newChain, err := o.chainSet.Default()
+				if err != nil || newChain == nil {
+					return err
+				}
```

Comment:
If err and newChain are both nil, then do we need to create an error to return?
```suggestion
				if err != nil {
					return err
				} else if newChain == nil {
					return errors.New()
				}
```

Reply:
Oops!